config/prod.exs

```diff
@@ -1,8 +1,13 @@
-# NOTE: This file should always be empty.
-#
-# Production-specific configuration should go in the special `MIX_ENV == "prod"`
-# block in `rel/config/config.exs` since we use an OTP release to deploy and
-# run the application in production.
-#
-# Configuration specified in this file would only be picked up at runtime when
-# running `mix phx.server` in production, which we never do.
+use Mix.Config
+
+# Configure endpoint
+config :elixir_boilerplate, ElixirBoilerplateWeb.Endpoint,
+  cache_static_manifest: "priv/static/cache_manifest.json",
```

Comment:
This reintroduces the bug fixed by #33 — The `cache_static_manifest` configuration on `ElixirBoilerplateWeb.Endpoint` must be set at _runtime_ not at _compile time_.

Reply:
After further investigation, `mix release` correctly generate a `sys.config` file in the OTP release with configuration from `config/*.exs`! Runtime configuration get merged in when the `Mix.Releases.Config.Providers.Elixir` runs at boot time to parse the `rel/config/runtime.exs` file that gest overlayed in the bundle.